README.md

```diff
@@ -133,7 +133,7 @@ bun run build
 Run the compiled server code. By default, this will start the REST API server on port 4000 and the MCP server on port 4040. The server will load the Malloy packages found in the submodules.
 
 ```bash
-bun run start
+NODE_ENV=production bun run start
```

Comment:
Oh I thought this was the default... Should move this into the start command in package.json.